app/controllers/api/automate_controller.rb

```diff
@@ -12,6 +12,21 @@ def show
       render_resource :automate, :name => "automate", :subcount => resources.count, :resources => resources
     end
 
+    def git_refresh_resource(type, id = nil, data = nil)
+      api_action(type, id) do |klass|
+        unless MiqRegion.my_region.role_active?("git_owner")
```

Comment:
Can we have this logic in model, maybe a helper method in GitBasedDomainImportService that we can call ?